distribution/docker/ddf/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version. 
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>ddf.distribution</groupId>
+        <artifactId>docker</artifactId>
+        <version>2.14.0-SNAPSHOT</version>
+    </parent>
+    <artifactId>ddf</artifactId>
+    <groupId>ddf.distribution.docker</groupId>
+    <packaging>pom</packaging>
+    <name>DDF ::  Distribution :: Docker :: DDF</name>
+
+    <properties>
+        <docker.ddf.base.version>2.14-alpine</docker.ddf.base.version>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.codice.ddf</groupId>
+            <artifactId>ddf</artifactId>
+            <version>${project.version}</version>
+            <type>zip</type>
+            <exclusions>
+                <exclusion>
+                  <groupId>*</groupId>
+                  <artifactId>*</artifactId>
+                </exclusion>
+            </exclusions>
+        </dependency>
+    </dependencies>
+
+
+    <profiles>
+        <profile>
+            <id>docker</id>
+            <activation>
+                <os>
+                    <family>unix</family>
```

Comment:
❓Does this mean that it won't be possible to use those images on Windows to run the itests?

Reply:
This is just a temporary measure for now. Once we start using containers during the itests we will probably want to update the usage of the docker maven plugin to create a `docker-machine` instance when there is no docker daemon running on the host. This would allow it to transparently work on all platforms